tech.json

```diff
@@ -77,12 +77,27 @@
   {
     "name": "canCrystalFlash",
     "requires": [
-      {"Missile": 2},
-      {"Super": 2},
-      {"PowerBomb": 3},
-      "Morph"
-    ],
-    "note": "FIXME: Needs ability for reader to count stuff, splitting on colon"
+      {"cost":[
+        {"ammo": [
+          {
+            "type": "Missile",
+            "count": 2
```

Comment:
???

Reply:
Went with the numbers that where here earlier since I didn't remember. Changing.

Reply:
Those were numbers of packs, because you need 10/10/11.